benchmarks/datasets/wikitext2_data.py

```diff
@@ -41,25 +41,21 @@ def batchify(data):
         batch_size = args.batch_size
         return _batchify(data, batch_size)
 
-    # TODO(anj-s): Both seq_len and batch size should be part of the golden config.
-    seq_len = 32
-    total_batch_size = seq_len * args.batch_size
+    total_batch_size = benchmark_config["seq_len"] * benchmark_config["batch_size"]
     train_dataloader = DataLoader(train_dataset, batch_size=total_batch_size, collate_fn=batchify)
     valid_dataloader = DataLoader(valid_dataset, batch_size=total_batch_size, collate_fn=batchify)
     test_dataloader = DataLoader(test_dataset, batch_size=total_batch_size, collate_fn=batchify)
     return len(vocab.stoi), train_dataloader, valid_dataloader, test_dataloader
 
 
-def get_synthetic_dataloaders(args):
+def get_synthetic_dataloaders(args, benchmark_config):
     """Return synthetic dataloaders for training, testing and validation."""
 
     def batchify(data):
         batch_size = args.batch_size
         return _batchify(data, batch_size)
 
-    # TODO(anj-s): Both seq_len and batch size should be part of the golden config.
-    seq_len = 32
-    total_batch_size = seq_len * args.batch_size
+    total_batch_size = benchmark_config["seq_len"] * benchmark_config["batch_size"]
```

Comment:
nit: we can dedup this and line 44 into a common function in case we change one place and forget another in the future.